### PR TITLE
Removes obsolete captain outfit

### PR DESCRIPTION
**Pull request by tf-4**
## About The Pull Request

Removes obsolete outfit

## Changelog
:cl:
del: A broken and obsolete captain's clothing item has been removed
/:cl:

### CODE REVIEW

**Review by axietheaxolotl — CHANGES_REQUESTED** [review]
We don't actually need this. The base Captain's jacket from tg can transform into the female one with an alt-click. Why these two separate ones exist I still don't know.

**Comment by GoldenAlpharex** [comment]
It is indeed not necessary, the standard parade uniform allows you to switch between them.
I can't believe you forgor 💀

**Comment by tf-4** [comment]
I don't play captain :)

File: _maps/map_files/Blueshift/BlueShift.dmm [hunk 1]
```diff
@@ -113656,8 +113656,6 @@
 /obj/item/clothing/under/rank/captain/suit,
 /obj/item/clothing/under/rank/captain/suit/skirt,
 /obj/item/clothing/under/rank/captain/parade,
-/obj/item/clothing/suit/armor/captain/parade/female,
-/obj/item/clothing/suit/armor/captain/parade,
```
Comment (GoldenAlpharex) [inline _maps/map_files/Blueshift/BlueShift.dmm:113660]: Was that second one also broken?

**Review by GoldenAlpharex — COMMENTED** [review]
I just need this answered before I can approve.

File: _maps/map_files/Blueshift/BlueShift.dmm (hunk 1 above)
Comment (tf-4) [reply _maps/map_files/Blueshift/BlueShift.dmm:113660]: it doesn't exist so yes

**Pull request merged**